labelspark/__init__.py

```diff
@@ -153,7 +152,7 @@ def bronze_to_silver(bronze_table):
                 "frameNumber"] = row.frameNumber  # need to store the unique framenumber identifier for video
         new_json.append(my_dictionary)
 
-    parsed_classifications = pd.DataFrame(new_json).to_spark()
+    parsed_classifications = spark.read.json(new_json)
```

Comment:
On pre Spark 3.2, does spark.read.json() achieve the same outcome as the koalas-based pd.DataFrame(new_json).to_spark() previously written? 

If not, we could also wrap this with a conditional check and use the old code if it needs Koalas: 

```
if needs_koalas: 
   parsed_classifications = pd.DataFrame(new_json).to_spark()
else: 
   parsed_classifications = spark.read.json(new_json)
```